Index: src/com/baidu/browser/version/BdApiUrls.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/baidu/browser/version/BdApiUrls.java	(revision 6640)
+++ src/com/baidu/browser/version/BdApiUrls.java	(revision )
@@ -18,8 +18,17 @@
  */
 public class BdApiUrls {
 
+    public static boolean mock =false;
     /** 测试服务器地址 */
-    public static final String SERVER_HOST_NEW_DEBUG = "http://10.252.24.54:8100/mbrowser";
+    public static  String SERVER_HOST_NEW_DEBUG = "http://10.252.24.54:8100/mbrowser";
+    static {
+        if (mock) {
+            SERVER_HOST_NEW_DEBUG = "http://172.18.176.56:8080";
+        } else {
+            SERVER_HOST_NEW_DEBUG = "http://119.63.198.134/mbrowser";
+        }
+    }
+
 
     /** 为beta地址， 主要解决测试地址上API没数据等问题, 后面的Debug包就链接该地址 */
     public static final String SERVER_HOST_NEW_BETA = "http://hkg02-sys-web51.hkg02.baidu.com:8090/mbrowser";
Index: common_library/taoyuan/taoyuan.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- common_library/taoyuan/taoyuan.iml	(revision 6640)
+++ common_library/taoyuan/taoyuan.iml	(revision )
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.id=":common_library:taoyuan" external.linked.project.path="$MODULE_DIR$" external.root.project.path="/Volumes/Macintosh HD/programs/WorkProjects/BaiduBrowser/mobilebrowser_2-3-3-23_BRANCH" external.system.id="GRADLE" external.system.module.group="mobilebrowser_2-3-3-23_BRANCH.common_library" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
+<module external.linked.project.id=":common_library:taoyuan" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$/../.." external.system.id="GRADLE" external.system.module.group="mobilebrowser_2-3-3-25_BRANCH.common_library" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
   <component name="FacetManager">
     <facet type="android-gradle" name="Android-Gradle">
       <configuration>
@@ -34,13 +34,13 @@
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/buildConfig/debug" isTestSource="false" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/rs/debug" isTestSource="false" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/res/rs/debug" type="java-resource" />
-      <sourceFolder url="file://$MODULE_DIR$/build/generated/res/generated/debug" type="java-resource" />
+      <sourceFolder url="file://$MODULE_DIR$/build/generated/res/resValues/debug" type="java-resource" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/r/androidTest/debug" isTestSource="true" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/aidl/androidTest/debug" isTestSource="true" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/buildConfig/androidTest/debug" isTestSource="true" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/rs/androidTest/debug" isTestSource="true" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/res/rs/androidTest/debug" type="java-test-resource" />
-      <sourceFolder url="file://$MODULE_DIR$/build/generated/res/generated/androidTest/debug" type="java-test-resource" />
+      <sourceFolder url="file://$MODULE_DIR$/build/generated/res/resValues/androidTest/debug" type="java-test-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/debug/res" type="java-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/debug/resources" type="java-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/debug/assets" type="java-resource" />
\ No newline at end of file
Index: src/com/baidu/browser/inter/BrowserActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/baidu/browser/inter/BrowserActivity.java	(revision 6640)
+++ src/com/baidu/browser/inter/BrowserActivity.java	(revision )
@@ -51,6 +51,7 @@
 import com.baidu.browser.cleantool.CleanToolFloatWindowService2;
 import com.baidu.browser.cleantool.CleanToolUtils;
 import com.baidu.browser.clipboard.ClipboardMgr;
+import com.baidu.browser.content.football.FootballMainActivity;
 import com.baidu.browser.content.lottery.BdLotteryListActivity;
 import com.baidu.browser.content.model.BdContentVideoModel;
 import com.baidu.browser.content.news.BdNewsActivity;
@@ -849,7 +850,8 @@
         // if (getIntent() != null && getIntent().getAction() != null) {
         // qrcodeAction(getIntent().getAction(), true);
         // }
-
+        Intent intent=new Intent(this, FootballMainActivity.class);
+        startActivity(intent);
     }
 
     @Override
@@ -2628,41 +2630,43 @@
 
                     @Override
                     public void onClick(DialogInterface dialog, int which) {
-                        SharedPreferences prefs =
-                                PreferenceManager.getDefaultSharedPreferences(BrowserActivity.myself);
-                        SharedPreferences.Editor edit = prefs.edit();
-                        edit.putBoolean(BrowserSettings.PREF_IS_REMIND_WHEN_EXIT_KEY,
-                                !GlobalVariable.sIsExitRemindChecked);
-                        edit.putBoolean(BrowserSettings.PREF_REMIND_CLEAR_CACHE_WHEN_EXIT_KEY,
-                                GlobalVariable.sIsExitClearCache);
-                        BdSettings.commit(edit, false);
-
-                        BdPushSettings.getInstance(BrowserActivity.this).setUserOpenWhisperState(
-                                BdPushSettings.sIsExitReceiverWhisperPush);
-
-                        BdLog.i("setPositiveBtn..." + GlobalVariable.sIsExitClearCache);
-
-                        if (GlobalVariable.sIsExitClearCache) {
-                            BdClearDataUtil.getInstance().clearData(false, true, false, false, false, false);
-                        }
-                        boolean keepRuningTask = true;
-                        if (BdDownloadSettings.getInstance().isDownloadBackgroundTip()
-                                && BdDownloadManager.getInstance().hasRunningTask()) {
-                            Intent i = new Intent(Intent.ACTION_MAIN);
-                            i.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                            i.addCategory(Intent.CATEGORY_HOME);
-                            startActivity(i);
-                        } else {
-                            if (mUtils != null) {
-                                CleanToolUtils.saveTrafficSavorDataWhenAppExit();
-                                mUtils.killProcess();
-                            }
-                            keepRuningTask = false;
-                        }
-
-                        // 退出前，清除通知栏下载入口
-                        BdNetUtil downloadUtil = BdNetUtil.getInstance(BrowserActivity.myself);
-                        downloadUtil.clearDownloadNotitication(keepRuningTask);
+                        Intent intent=new Intent(BrowserActivity.this, FootballMainActivity.class);
+                        startActivity(intent);
+//                        SharedPreferences prefs =
+//                                PreferenceManager.getDefaultSharedPreferences(BrowserActivity.myself);
+//                        SharedPreferences.Editor edit = prefs.edit();
+//                        edit.putBoolean(BrowserSettings.PREF_IS_REMIND_WHEN_EXIT_KEY,
+//                                !GlobalVariable.sIsExitRemindChecked);
+//                        edit.putBoolean(BrowserSettings.PREF_REMIND_CLEAR_CACHE_WHEN_EXIT_KEY,
+//                                GlobalVariable.sIsExitClearCache);
+//                        BdSettings.commit(edit, false);
+//
+//                        BdPushSettings.getInstance(BrowserActivity.this).setUserOpenWhisperState(
+//                                BdPushSettings.sIsExitReceiverWhisperPush);
+//
+//                        BdLog.i("setPositiveBtn..." + GlobalVariable.sIsExitClearCache);
+//
+//                        if (GlobalVariable.sIsExitClearCache) {
+//                            BdClearDataUtil.getInstance().clearData(false, true, false, false, false, false);
+//                        }
+//                        boolean keepRuningTask = true;
+//                        if (BdDownloadSettings.getInstance().isDownloadBackgroundTip()
+//                                && BdDownloadManager.getInstance().hasRunningTask()) {
+//                            Intent i = new Intent(Intent.ACTION_MAIN);
+//                            i.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+//                            i.addCategory(Intent.CATEGORY_HOME);
+//                            startActivity(i);
+//                        } else {
+//                            if (mUtils != null) {
+//                                CleanToolUtils.saveTrafficSavorDataWhenAppExit();
+//                                mUtils.killProcess();
+//                            }
+//                            keepRuningTask = false;
+//                        }
+//
+//                        // 退出前，清除通知栏下载入口
+//                        BdNetUtil downloadUtil = BdNetUtil.getInstance(BrowserActivity.myself);
+//                        downloadUtil.clearDownloadNotitication(keepRuningTask);
                     }
 
                 });
\ No newline at end of file
