Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- build.gradle	(revision 5993)
+++ build.gradle	(revision )
@@ -126,6 +126,6 @@
 
 dependencies {
     compile project(':common_library:taoyuan')
-
     compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile files('libs/deeplink.1.5.jar')
 }
Index: src/com/baidu/browser/inter/BrowserActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/baidu/browser/inter/BrowserActivity.java	(revision 6078)
+++ src/com/baidu/browser/inter/BrowserActivity.java	(revision )
@@ -50,6 +50,8 @@
 import com.baidu.browser.cleantool.CleanSuspendMgr;
 import com.baidu.browser.cleantool.CleanToolFloatWindowManager;
 import com.baidu.browser.cleantool.CleanToolFloatWindowService;
+import com.baidu.browser.cleantool.CleanToolMainActivity;
+import com.baidu.browser.cleantool.CleanToolMainDialog;
 import com.baidu.browser.cleantool.CleanToolUtils;
 import com.baidu.browser.clipboard.ClipboardMgr;
 import com.baidu.browser.content.lottery.BdLotteryListActivity;
@@ -634,6 +636,8 @@
     public void onCreate(Bundle savedInstanceState) {
         BdUtils.openStrictMode();
         super.onCreate(savedInstanceState);
+
+        CleanToolMainDialog.showCleanToolDialog(this);
         myself = this;
         if (BdDebug.DEBUG_TIME_LAUNCH) {
             BdLog.e("--------onCreate() start");
\ No newline at end of file
Index: src/com/baidu/browser/inter/BdApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/baidu/browser/inter/BdApplication.java	(revision 5993)
+++ src/com/baidu/browser/inter/BdApplication.java	(revision )
@@ -1,7 +1,5 @@
 package com.baidu.browser.inter;
 
-import java.io.File;
-
 import android.app.Application;
 import android.content.Context;
 
@@ -25,6 +23,8 @@
 import com.nostra13.universalimageloader.core.ImageLoaderConfiguration;
 import com.nostra13.universalimageloader.core.assist.QueueProcessingType;
 
+import java.io.File;
+
 /**
  * 应用程序
  */
@@ -42,24 +42,32 @@
         }
     }
 
-    /** Singleton Instance */
+    /**
+     * Singleton Instance
+     */
     private static BdApplication sInstance;
-    /** 调试信息 */
+    /**
+     * 调试信息
+     */
     private BdDebug mDebug;
 
-    /** App Context : added by lijun04 **/
+    /**
+     * App Context : added by lijun04
+     **/
     public static Context mAppContext;
 
     /**
      * ImageLoader log 调试开关
      */
     private static boolean IMAGELOADER_DEBUG_MODE = BdDebug.DEBUG_MODE && false;
-    /** corruption elapse : added by lijun04 **/
+    /**
+     * corruption elapse : added by lijun04
+     **/
     public static long elapse = System.currentTimeMillis();
 
     /**
      * GetInstance
-     * 
+     *
      * @return {@link BdApplication}
      */
     public static BdApplication getInstance() {
@@ -68,7 +76,7 @@
 
     /**
      * SetInstance
-     * 
+     *
      * @param aInstance {@link BdApplication}
      */
     private static void setInstance(BdApplication aInstance) {
@@ -77,7 +85,7 @@
 
     /**
      * GetAppContext
-     * 
+     *
      * @return
      */
     public static Context getAppContext() {
@@ -86,7 +94,7 @@
 
     /**
      * SetAppContext
-     * 
+     *
      * @param aContext
      */
     private static void setAppContext(Context aContext) {
@@ -153,9 +161,9 @@
             // asset里预置的九宫格图片全部换成了新图标，由于ImageLoader在SDcard中缓存了旧的图标，为了防止使用缓存的旧图标，这里需要清理缓存
             if (BdVersionController.getInstance().isOuterVersionUpgrade()
                     && (BdVersionController.getInstance().getLastOuterVersion() != null
-                            && BdVersionController.getInstance().getLastOuterVersion().length() > 1
-                            && BdVersionController.getInstance().getLastOuterVersion().charAt(0) < '5'
-                            && BdVersionController.getInstance().getLastOuterVersion().charAt(1) == '.')) {
+                    && BdVersionController.getInstance().getLastOuterVersion().length() > 1
+                    && BdVersionController.getInstance().getLastOuterVersion().charAt(0) < '5'
+                    && BdVersionController.getInstance().getLastOuterVersion().charAt(1) == '.')) {
                 new Thread() {
                     @Override
                     public void run() {
@@ -167,7 +175,7 @@
             configBuilder =
                     configBuilder
                             .threadPriority(Thread.NORM_PRIORITY - 2)
-                            // .denyCacheImageMultipleSizesInMemory()
+                                    // .denyCacheImageMultipleSizesInMemory()
                             .diskCacheFileNameGenerator(new Md5FileNameGenerator())
                             .diskCache(diskCache)
                             .memoryCacheSize(5 * 1024 * 1024).threadPoolSize(3)
@@ -185,7 +193,9 @@
         return (BdDebugHandler) mDebug.getHandler();
     }
 
-    /** get instance of BrowserActivity : added by lijun04 **/
+    /**
+     * get instance of BrowserActivity : added by lijun04
+     **/
     public static BrowserActivity getBrowserActivity() {
         return BrowserActivity.myself;
     }
